custom_components/localtuya/pytuya/__init__.py

```diff
@@ -851,8 +861,18 @@ async def detect_available_dps(self):
         # list of available dps experience shows that the dps available are usually
         # in the ranges [1-25] and [100-110] need to split the bruteforcing in
```

Comment:
Could you adjust this comment, now that you've expanded the list?

Reply:
Will update